### PR TITLE
Update api-auth docs

### PR DESCRIPTION
**Pull request by cabralmartin**
- Updated documentation for `api-auth`
- Removed all flows under `/i` (deprecated)
- Removed all flows other than `client credentials`
- Updated dashboard screenshots to reflect latest changes
- Removed section `configuring scopes` since this is now available thru the dashboard
- Removed all examples other than `server-to-server`

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/api-auth/config/using-the-auth0-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+1. Open the Auth0 Dashboard and browse to the Clients section.
+
+2. Here you will create a new client of type Non Interactive for each of the applications that will consume the API you want to generate access tokens for.
+
+3. Navigate to the API section and create a new "API" by entering a friendly name and identifier for it (ideally this would be the public endpoint of API). This API will represent your resource server.
+The selection of the Signing Algorithm will dictate how the API will validate the access tokens it receives
+  * HS256 (symmetric): signed using the resource server's signing secret
+  * RS256 (asymmetric): signed using Auth0's private key for your account. Verification is done using the corresponding public key, which can be found at the following standard [JWKS (JSON Web Key set)](https://self-issued.info/docs/draft-ietf-jose-json-web-key.html) URL: https://${account.namespace}/.well-known/jwks.json
+
+![](/media/articles/api-auth/apis-create.png)
```
Comment (Catografix) [inline articles/api-auth/config/using-the-auth0-dashboard.md]: To have the ordered list numbering continue after an image, indent the image tags 2 spaces.

File: articles/api-auth/config/using-the-management-api.md [hunk 2]
```diff
@@ -1,16 +1,14 @@
...
 In case you don't want to use the Auth0 Dashboard you can perform all the same operations using Auth0's Management API.
 
 Keep in mind that you will need the following items:
 - APIv2 token with the following scopes
   - create:resource_server
   - create:client_grant
-- Client Id and Secret for the client Application which has to be already created and visible in your [Auth0 dashboard](https://manage.auth0.com/)
-
-> Note: You can create your API token using the [API v2 Explorer](https://auth0.com/docs/api/v2)
+- Client Id and Secret for the client application which has to be already created and visible in your [Auth0 dashboard](https://manage.auth0.com/)
```
Comment (Catografix) [inline articles/api-auth/config/using-the-management-api.md]: `[Auth0 dashboard](${uiURL}).`

File: articles/api-auth/config/using-the-auth0-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+---
+
+# API Authorization: Using the Auth0 Dashboard
+<%=include('../_preview-warning') %>
+
+1. Open the [Auth0 Dashboard](${uiURL}) and browse to the Clients section.
+
+2. Create a new client of type **Non Interactive** for each of the applications that will consume the API you want to generate access tokens for.
+
+3. Navigate to the API section and create a new API by entering a friendly name and identifier (this would be the public endpoint of the API). This API will represent your resource server.
```
Comment (cabralmartin) [inline articles/api-auth/config/using-the-auth0-dashboard.md]: Ideally the identifier should be the public endpoint of the API but this is not mandatory. It is ok to submit any valid urn as identifier

**Pull request merged**